pkg/integration/tests/patch_building/move_range_to_index.go

```diff
@@ -0,0 +1,70 @@
+package patch_building
```

Comment:
I would keep this test but delete the other ones: given that this PR only really affects logic around adding files to the custom patch, but not logic around what we do with that custom patch, one test will suffice to know that it's working properly (and we don't want to have to maintain too many integration tests)